rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/InstallPackageCommand.java

```diff
@@ -76,6 +76,8 @@ public void execute(ApplicationContext currentContext) throws CLIException {
 
             schedulerProperties = retrieveSchedulerProperties(currentContext, scheduler);
 
+            schedulerProperties = addSessionIdToSchedulerProperties(currentContext, schedulerProperties);
```

Comment:
no need to assign schedulerProperties = ...  if you pass schedulerPropertiesas parameters. It's a p[ass by reference argument